CONTRIBUTING.md

```diff
@@ -1,88 +1,206 @@
-
-## Quality insurance for pull requests for XAML
-
-We encourage developers to follow the following guidances when submitting pull requests for XAML:
-
-* Your XAML must be usable and efficient with keyboard only.
-* Tab order must be logical.
-* Focused controls must be visible.
-* Action must be triggered when hitting Enter key.
-* Do not use custom colors but instead rely on theme colors so high contrasts themes can be used with your control.
-* Add `AutomationProperties.Name` on all controls to define what the controls purpose (Name is minimum, but there are some other things too that can really help the screen reader).
-* Don't use the same Name on two different elements unless they have different control types.
-* Use Narrator Dev mode (Launch Narrator `WinKey+Enter`, then `CTRL+F12`) to test the screen reader experience. Is the information sufficient, meaningful and helps the user navigate and understand your control?
-
-You can find more information about these topics [here](https://blogs.msdn.microsoft.com/winuiautomation/2015/07/14/building-accessible-windows-universal-apps-introduction). This is to help as part of our effort to building accessible templates from the start.
-
-## General rules
-
-* DO NOT require that users perform any extensive initialization before they can start programming basic scenarios.
-* DO NOT use regions. DO use partial classes instead.
-* DO NOT seal controls.
-* DO NOT use verbs that are not already used like fetch.
-* DO NOT return true or false to give sucess status. Throw exceptions if there was a failure.
-* DO provide good defaults for all values associated with parameters, options, etc.
-* DO ensure that APIs are intuitive and can be successfully used in basic scenarios without referring to the reference documentation.
-* DO communicate incorrect usage of APIs as soon as possible.
-* DO design an API by writing code samples for the main scenarios. Only then, you define the object model that supports those code samples.
-* DO declare static dependency properties at the top of their file.
-* DO use extension methods over static methods where possible.
-* DO use verbs like GET.
-
-## Naming conventions
-
-* We are following the coding guidelines of [.NET Core coding style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).
-
-## Documentation
-
-* DO NOT expect that your code is so well designed that it needs no documentation. No code is that intuitive.
-* DO provide great documentation with all new features and code.
-* DO use readable and self-documenting identifier names.
-* DO use consistent naming and terminology.
-* DO provide strongly typed APIs.
-* DO use verbose identifier names.
-
-## Files and folders
-
-* DO associate no more than one class per file.
-* DO use folders to group classes based on features.
+* [Issues](https://github.com/microsoft/TemplateStudio/issues)
+* [Discussions](https://github.com/microsoft/TemplateStudio/discussions)
```

Comment:
Discussions aren't enabled for the repo. Enable them in settings (I don't have permission) or remove this.

Reply:
Enabled.